js/screens/MatchScreenLoading/MatchScreenLoadingContainer.js

```diff
@@ -2,12 +2,18 @@ import React, { Component } from "react";
 import { connect } from "react-redux";
 import PropTypes from "prop-types";
 
+import { createTeamAndAddUser } from "../../redux/modules/teams";
 import MatchScreenLoading from "./MatchScreenLoading";
 
 class MatchScreenLoadingContainer extends Component {
   static navigationOptions = {
     header: null
   };
+  constructor() {
+    super();
+    this.createNewTeamWithUser = this.createNewTeamWithUser.bind(this);
```

Comment:
since you are using an arrow function, you do need to bind it. Please remove this line.